packages/@react-aria/slider/src/useSlider.ts

```diff
@@ -162,6 +176,11 @@ export function useSlider(
       onMouseDown(e: React.MouseEvent<HTMLElement>) { onDownTrack(e, undefined, e.clientX, e.clientY); },
       onPointerDown(e: React.PointerEvent<HTMLElement>) { onDownTrack(e, e.pointerId, e.clientX, e.clientY); },
       onTouchStart(e: React.TouchEvent<HTMLElement>) { onDownTrack(e, e.changedTouches[0].identifier, e.changedTouches[0].clientX, e.changedTouches[0].clientY); }
-    }, moveProps)
+    }, moveProps),
+    outputProps: {
+      htmlFor: state.values.map((_, index) => getSliderThumbId(state, index)).join(' '),
+      'aria-labelledby': labelProps.id,
+      'aria-live': 'off'
+    }
```

Comment:
Just to make sure:
when you say `Uses a single <output> element for the whole value label given that we support custom value formats through valueLabel. This loses clicking on the individual values to focus the thumb, but I think this should hopefully be ok since it was kind of undiscoverable anyway.`
does this mean:
-  clicking on the `<output>` shouldn't focus the thumb at all 
OR
- that clicking the individual values in the output (e.g. in a range slider) should focus the same thumb, we are just losing the individual thumb focus behavior that was added previously

Reply:
I don't believe clicking does anything anymore.